magic/core/src/main/java/link/magic/android/core/relayer/message/MagicRequestData.kt

```diff
@@ -3,6 +3,7 @@ import androidx.annotation.Keep
 import link.magic.android.core.provider.RequestForSerialization
 
 @Keep
-internal class RequestData(val payload: RequestForSerialization<*>, outboundMessageType: OutboundMessageType, encodedParams: String) {
+internal class MagicRequestData(val payload: RequestForSerialization<*>, outboundMessageType: OutboundMessageType, encodedParams: String, val rt: String? = null) {
```

Comment:
Why is `rt` being passed here and not used?

Reply:
It's the placeholder for the refresh token in the future. As we enable generating a dpop token in mobile SDK, we might consider implementing custom sessions for mobile platforms.